static/templates/admin_user_group_list.hbs

```diff
@@ -21,7 +22,7 @@
         <div class="input" contenteditable="true" data-placeholder="{{t 'Add member...' }}"></div>
     </div>
     <p class="save-instructions">
-        {{t "Click outside the input box to save. We'll automatically notify anyone that was added or removed."}}
+        {{t "Save/Discard changes of one USER GROUP at a time."}}
```

Comment:
I don't think we need the caps here. Also, it's worth keeping a part of the original text which says "we'll notify anyone added".